Formula/molecule.rb

```diff
@@ -117,8 +131,9 @@ class Molecule < Formula
   def install
     virtualenv_install_with_resources
 
-    site_packages = Language::Python.site_packages("python3.10")
-    %w[cookiecutter jsonschema].each do |package_name|
+    site_packages = Language::Python.site_packages("python3.11")
+    # %w[cookiecutter jsonschema].each do |package_name|
```

Comment:
Why not `jsonschema`? Also, we do not keep old code as comments, but rather a short explanation.